src/main/java/org/quiltmc/installer/action/InstallServer.java

```diff
@@ -255,7 +255,8 @@ public void run(Consumer<MessageType> statusTracker) {
 	private static CompletableFuture<Path> downloadLibrary(String name, String url) {
 		return CompletableFuture.supplyAsync(() -> {
 			try {
-				Path path = Files.createTempFile(name, null);
+				String shortName = name.split(":")[1];
+				Path path = Files.createTempFile(shortName, null);
```

Comment:
Could you also include the artifact version? This probably will be temporary, as I will probably make it cache in the future.

Reply:
Done, although now I'm not sure if `shortName` is a good variable name for this